src/index.html

```diff
@@ -11,7 +11,179 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="box-container">
+        <legend>Personal information</legend>
+        <label
+        class="form-label"
+        for="surname">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            id="surname"
+            minlength="2"
+            maxlength="30"
+            required
+          >
+        </label>
+
+        <label class="form-label" for="name">
+          Name:
+          <input
+            type="text"
+            autocomplete="off"
+            id="name"
+            minlength="2"
+            maxlength="30"
+            required
+          >
+        </label>
+
+        <label class="form-label" for="age">
+          How old are You?
+          <input
+            type="number"
+            value="12"
+            id="age"
+            step="1"
+            min="1"
+            max="100"
+            required
+          >
+        </label>
+
+        <label class="form-label" for="date-of-birth">
+          Full date of birth:
+          <input
+            type="date"
+            name="data-of-birth"
+            required
+          >
+        </label>
+
+        <label class="form-label" for="agree">
+          I accept the term of the agreement
+          <input type="checkbox" id="agree">
+        </label>
+      </fieldset>
+
+      <fieldset class="box-container">
+        <legend>Registration</legend>
+        <label class="form-label" for="email">
+          E-mail:
+          <input
+            type="email"
+            id="email"
+            placeholder="email@example.com"
+            name="email"
+            required
+          >
+        </label>
+
+        <label class="form-label" for="password">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="8"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="box-container">
+        <legend>An interesting fact about you!</legend>
+        <div class="cat-container">
+          <label class="cats" for="fact">Do you love cats?</label>
```

Comment:
what should this label do?
there's no element with `id="fact"`
```suggestion
          Do you love cats?
```